ui/pages/create-password-form/index.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState, useContext } from 'react';
+import {
+  IconColor,
+  JustifyContent,
+  AlignItems,
+  TextVariant,
+  TextColor,
+  BlockSize,
+  FlexDirection,
+  Display,
+  BackgroundColor,
+  BorderRadius,
+} from '../../helpers/constants/design-system';
+import {
```

Comment:
Since it's a new component. We should use design-system react library if possible